2025-08-25 17:48:09,795	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-25 17:48:09,939	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 1825
2025-08-25 17:48:10,474	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-25 17:48:10,474	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-25 17:48:10,475	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-25 17:48:10,476	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-25 17:48:10,592	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-25 17:48:10,697	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-25 17:48:10,710	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 10:23:25,017	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 10:23:25,215	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6443
2025-08-26 10:23:26,189	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 10:23:26,190	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 10:23:26,190	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 10:23:26,190	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 10:23:26,301	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 10:23:26,401	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 10:23:26,401	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 10:28:43,767	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 10:28:43,893	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6124
2025-08-26 10:28:44,364	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 10:28:44,364	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 10:28:44,365	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 10:28:44,365	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 10:28:44,487	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 10:28:44,563	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 10:28:44,563	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 10:34:42,555	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 10:34:43,056	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 5765
2025-08-26 10:34:43,874	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 10:34:43,875	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 10:34:43,875	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 10:34:43,875	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 10:34:44,082	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 10:34:44,282	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 10:34:44,282	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 10:42:46,377	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 10:42:46,532	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 5282
2025-08-26 10:42:46,941	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 10:42:46,941	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 10:42:46,941	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 10:42:46,942	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 10:42:47,039	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 10:42:47,109	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 10:42:47,109	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 10:58:20,279	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 10:58:20,485	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 4348
2025-08-26 10:58:20,952	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 10:58:20,952	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 10:58:20,952	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 10:58:20,952	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 10:58:21,070	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 10:58:21,181	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 10:58:21,181	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 11:05:05,397	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 11:05:05,528	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 3943
2025-08-26 11:05:06,476	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 11:05:06,477	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 11:05:06,477	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 11:05:06,477	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 11:05:06,581	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 11:05:06,657	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 11:05:06,658	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 11:13:52,545	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 11:13:52,691	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 3415
2025-08-26 11:13:53,154	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 11:13:53,155	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 11:13:53,155	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 11:13:53,155	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 11:13:53,254	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 11:13:53,350	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 11:13:53,350	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 14:52:02,523	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 14:52:02,722	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 5050
2025-08-26 14:52:03,365	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 14:52:03,366	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 14:52:03,366	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 14:52:03,366	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 14:52:03,624	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 14:52:03,722	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 14:52:03,723	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 14:56:43,824	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 14:56:43,933	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 4769
2025-08-26 14:56:44,360	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 14:56:44,361	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 14:56:44,362	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 14:56:44,364	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 14:56:44,467	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 14:56:44,608	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 14:56:44,608	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 14:57:57,912	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '998073cb6a0b92351a931a62bc88708c'}, trace_id:998073cb6a0b92351a931a62bc88708c
2025-08-26 14:57:58,136	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'f2d96fafc5ad1cdc7146c3d3e27aacd9'}, trace_id:f2d96fafc5ad1cdc7146c3d3e27aacd9
2025-08-26 14:57:58,334	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'c98ae24361c28e0c75079d54ca403dd9'}, trace_id:c98ae24361c28e0c75079d54ca403dd9
2025-08-26 14:57:58,539	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'b13d53b2afc09da854098d01d701abb0'}, trace_id:b13d53b2afc09da854098d01d701abb0
2025-08-26 14:57:58,718	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'ef51c0e3cd5cdadd8f4b3266bc59531b'}, trace_id:ef51c0e3cd5cdadd8f4b3266bc59531b
2025-08-26 14:57:59,345	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '786357a4b3d1ee2bfbbaddcb2fce9394'}, trace_id:786357a4b3d1ee2bfbbaddcb2fce9394
2025-08-26 14:57:59,612	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '829fa7d703b6d09af3f8159f81274316'}, trace_id:829fa7d703b6d09af3f8159f81274316
2025-08-26 14:58:00,137	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '0342b0ae49f24bd5ebede951543d7df8'}, trace_id:0342b0ae49f24bd5ebede951543d7df8
2025-08-26 14:58:00,369	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '68c318ae3592aaa3551a830b8d263574'}, trace_id:68c318ae3592aaa3551a830b8d263574
2025-08-26 14:58:00,820	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'cc75113ddc907f59c8e4140d710c2733'}, trace_id:cc75113ddc907f59c8e4140d710c2733
2025-08-26 14:58:01,052	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '13799f9cd0aea5bdca75b3a756e81975'}, trace_id:13799f9cd0aea5bdca75b3a756e81975
2025-08-26 15:03:04,804	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 15:03:05,228	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 4388
2025-08-26 15:03:06,170	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 15:03:06,170	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 15:03:06,170	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 15:03:06,171	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 15:03:06,505	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 15:03:06,662	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 15:03:06,663	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 15:03:32,480	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'aaa0f1310557a8d284b5240981b9ef86'}, trace_id:aaa0f1310557a8d284b5240981b9ef86
2025-08-26 15:03:32,665	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '9fc1847201bfc6a9d6bf3f4c8f5642e4'}, trace_id:9fc1847201bfc6a9d6bf3f4c8f5642e4
2025-08-26 15:03:32,861	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '9cce7637d38838bcc9fe31dd0872eb0c'}, trace_id:9cce7637d38838bcc9fe31dd0872eb0c
2025-08-26 15:03:33,051	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '9c38bf97759645c44780d0efc23620c9'}, trace_id:9c38bf97759645c44780d0efc23620c9
2025-08-26 15:03:33,263	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '109d6bc82fe0bced2228ac6821eb1bdb'}, trace_id:109d6bc82fe0bced2228ac6821eb1bdb
2025-08-26 15:03:33,377	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '3fe64a57854f0285cc4b9ea58de50905'}, trace_id:3fe64a57854f0285cc4b9ea58de50905
2025-08-26 15:03:33,586	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'c49e143d8f764299d3c6db3a644c0aff'}, trace_id:c49e143d8f764299d3c6db3a644c0aff
2025-08-26 15:03:33,707	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'e40378a64848aeb1f2beeb07a457d882'}, trace_id:e40378a64848aeb1f2beeb07a457d882
2025-08-26 15:03:33,926	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '99636806e99f074a678405279b11ffa8'}, trace_id:99636806e99f074a678405279b11ffa8
2025-08-26 15:03:34,090	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': 'fa27168a6f64bb5909b64210363d43c2'}, trace_id:fa27168a6f64bb5909b64210363d43c2
2025-08-26 15:03:34,273	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/2060668842382180072/roles, 错误代码: 400, 返回内容: {'message': '频道末对机器人末授权', 'code': 11264, 'err_code': 40012009, 'trace_id': '0fea9792da7e1897a5f9397e6cc244b8'}, trace_id:0fea9792da7e1897a5f9397e6cc244b8
2025-08-26 15:07:45,549	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-08-26 15:07:45,648	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 4108
2025-08-26 15:07:46,055	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-08-26 15:07:46,055	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 15:07:46,056	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 15:07:46,056	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 15:07:46,130	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-08-26 15:07:46,214	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「竹涧聆虚」启动成功！
2025-08-26 15:07:46,214	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 15:08:03,094	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153827/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:a094a04e556ab618f55453284ae26d9e
2025-08-26 15:08:04,370	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153828/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:af8cd3fd836f87ce123be10e5b08ba49
2025-08-26 15:09:55,408	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153827/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:50c45427ba3dae951c2a2b3376cbc726
2025-08-26 15:09:55,702	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:75d98e11dddfa67a603c0cdb936e9306
2025-08-26 15:09:56,019	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:8269d1f74604c6be81272c34e19a8905
2025-08-26 15:09:56,323	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153828/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:422f5624d0bef7f75bcb7c22dab2678c
2025-08-26 15:15:07,802	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:8761584c099e49e058fea2265e3b42f1
2025-08-26 15:15:08,162	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:623ab9c6d7afada000d6f8c53930409b
2025-08-26 15:15:08,515	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:62571ec5b6ca22efbc11eb5e8afa65d4
2025-08-26 15:19:21,786	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:9ff53dd6118d835a102f2e2f3c9c81eb
2025-08-26 15:24:36,061	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:3c39c13c8fa89a8ef06245b92932e615
2025-08-26 15:24:36,422	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:0eeb4932519d8f08d51b0fc64fbeccd9
2025-08-26 15:24:36,802	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:63eff68250addb127166bbcc349e5685
2025-08-26 15:26:11,560	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:bc6af9a213c2970a1ab71b95e0d30083
2025-08-26 15:26:11,892	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:5851250c530b6018bb601b1a11b94835
2025-08-26 15:28:03,974	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:4676027f962de1dbaea70ab07fbec975
2025-08-26 15:28:04,449	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:7e45caa5c69679800d398f5c6a1afaae
2025-08-26 15:28:05,243	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153826/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:c094f84c1f82a9faa4a0a1d3b939bf3c
2025-08-26 15:31:59,042	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:86a9c52576a0e7520f85ecb72855ea35
2025-08-26 15:31:59,411	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:f8c0569b47184c1520dac06cd9f77a11
2025-08-26 15:31:59,720	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:1b26b8cd59e222cd1a6468dac0b1504a
2025-08-26 15:32:00,061	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:7ac33938ddb3ce9f0f89d469268b2d8a
2025-08-26 15:35:26,176	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:6e299c1f46510ee8fbb4e896d982a46e
2025-08-26 15:35:26,554	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:5eaa82ab43fdd244e867a8d74bac116f
2025-08-26 15:35:26,896	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:72abf863731967f7f8db0c327e7ecc92
2025-08-26 15:35:27,335	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153826/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:52255c6599376206c56fb8aa63753628
2025-08-26 15:35:27,678	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:7f07a9b98ec12ee2436e742fd15e7f81
2025-08-26 15:35:27,974	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:d7b49e5a820782bf11b0f4977a0a53b9
2025-08-26 15:35:28,311	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:a068481c02d554a2fd26b9739ebe3567
2025-08-26 15:35:28,608	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153825/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:56c880e2698a29492b006e872ecaca1f
2025-08-26 15:37:23,811	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:c9b63288fa03859a53bc93ce2709e96b
2025-08-26 15:37:24,184	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:a888d0c01e3906a5e235360b1bb9718e
2025-08-26 15:37:24,568	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:e753c6e64d3ef708a77600ac92f3d7ab
2025-08-26 15:37:24,899	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153825/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:85308eb07c1d4536d86abe526d62f4cb
2025-08-26 15:37:25,237	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:96c7a8a3a14190a2386d7d43814d19be
2025-08-26 15:37:46,128	[INFO]	(gateway.py:56)on_closed	[botpy] 关闭, 返回码: 4009, 返回信息: Session timed out
2025-08-26 15:37:46,128	[INFO]	(gateway.py:134)ws_connect	[botpy] ws关闭, 停止接收消息!
2025-08-26 15:37:51,132	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-08-26 15:37:51,133	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-08-26 15:37:51,133	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-08-26 15:37:51,240	[INFO]	(gateway.py:175)ws_resume	[botpy] 重连启动...
2025-08-26 15:37:51,305	[INFO]	(gateway.py:90)on_message	[botpy] 机器人重连成功! 
2025-08-26 15:37:51,305	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-08-26 15:38:55,915	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:dc57882234201168eb10e2498102c1b0
2025-08-26 15:38:56,235	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153825/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:0a756bbc0dc4f3817a358da7185469d6
2025-08-26 15:40:38,403	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:18214ecd933d3668cf2b89ed000be012
2025-08-26 15:40:39,762	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:8eec92a00c960a9ac0328594c736db0d
2025-08-26 15:40:40,120	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:eb6538244f453665af44922a33822ba2
2025-08-26 15:40:40,954	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153826/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:29a38be77668cb0e419892d302e3a563
2025-08-26 15:40:41,293	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:40ae1bbce616f94745d34da0be507e58
2025-08-26 15:40:41,598	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:7185aac4f698af4b3545bb005ce684a5
2025-08-26 15:40:41,961	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:97293ec727d1252ed0b2e21e6d8f9b46
2025-08-26 15:49:56,522	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153824/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:a3d97759c4f05ad85d90cd22fb67e6cb
2025-08-26 15:49:56,942	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:180a56cc6a69122c35946c9549616d97
2025-08-26 15:49:57,316	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153823/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:40a9b36696a3b6d4670d3e3ca5ff57c6
2025-08-26 15:49:57,627	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153820/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:22aad05c17862b524bba684f90b600a7
2025-08-26 15:49:57,858	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153825/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:fc03cec42c32d740387a73e148d858fb
2025-08-26 15:49:58,321	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153819/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:d57e28bb690cd1e76c99734f72ad70c4
2025-08-26 15:49:58,638	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153821/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:31a5aafc859c4b0b1b6387789ad3b83e
2025-08-26 15:52:41,944	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153826/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:e48ec2d738b0d0eb2851780ff66887ae
2025-08-26 15:52:42,326	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/659515934/roles/23153822/permissions, 错误代码: 500, 返回内容: {'code': 301003, 'message': 'put channel permission error'}, trace_id:11fdd80d3f5f3bd1568b1b58773a4339
